armi/cases/case.py

```diff
@@ -810,55 +823,55 @@ def copyInterfaceInputs(
                     )
             label = key.name
 
-            newSettings[label] = []
+            newFiles = []
             for f in files:
-                globFilePaths = None
-                path = pathlib.Path(f)
-                if path.is_absolute() and path.exists() and path.is_file():
-                    # Path is absolute, no settings modification or filecopy needed
-                    pass
-                else:
-                    # Path is either relative or includes a wildcard
-                    if not (path.exists() and path.is_file()):
-                        runLog.extra(
-                            f"Input file for `{label}` setting could not be resolved "
-                            f"with the following file path: `{path}`. Checking for "
-                            f"file at path `{sourceDirPath}`."
-                        )
-
-                    # Attempt to find relative path file
-                    sourceFullString = os.path.join(sourceDirPath, f)
-                    sourceFullPath = pathlib.Path(sourceFullString)
-                    if not os.path.exists(sourceFullPath):
-                        runLog.extra(
-                            f"Input file for `{label}` setting could not be resolved "
-                            f"with the following file path: `{sourceFullPath}`. Checking "
-                            f"for wildcards."
-                        )
-                        # Attempt to capture file paths from wildcards
-                        globFilePaths = [
-                            pathlib.Path(os.path.join(sourceDirPath, g))
-                            for g in glob.glob(sourceFullString)
-                        ]
-                        if len(globFilePaths) == 0:
-                            runLog.warning(
-                                f"No input files for `{label}` setting could be resolved "
-                                f"with the following file path: `{sourceFullPath}`."
-                            )
+                WILDCARD = False
+                RELATIVE = False
+                if "*" in f:
+                    WILDCARD = True
+                if ".." in f:
+                    RELATIVE = True
 
-                    # Finally, copy + update settings according to file path type
-                    if not globFilePaths:
-                        destFilePath = copyInputsHelper(label, sourceFullPath, destPath)
-                        # Some settings are a single filename. Others are lists of files.
-                        # Either overwrite the empty list at the top of the loop, or
-                        # append to it.
-                        if len(files) == 1:
-                            newSettings[label] = str(destFilePath)
-                        else:
-                            newSettings[label].append(str(destFilePath))
+                path = pathlib.Path(f)
+                if WILDCARD == False and RELATIVE == False:
```

Comment:
I think `pylint` would prefer it if this read:

`if not WILDCARD and not RELATIVE:`